definitions/features/installer.rb

```diff
@@ -0,0 +1,75 @@
+class Features::Installer < ForemanMaintain::Feature
+  metadata do
+    label :installer
+
+    confine do
+      find_package('foreman-installer') ||
+        find_package('katello-installer') ||
+        find_package('capsule-installer')
+    end
+  end
+
+  def initialize
+    @installer_type = if find_package('foreman-installer')
+                        :scenarios
+                      elsif find_package('katello-installer')
+                        :katello
+                      elsif find_package('capsule-installer')
```

Comment:
Upstream foreman-proxy-with-contents will use a `foreman-proxy-content` package, shouldn't this be accounted for? i.e.
```
[vagrant@bacon-proxy ~]$ rpm -qa | grep proxy
foreman-proxy-content-3.7.0-1.nightly.el7.noarch
bacon-proxy.jomitsch.example.com-apache-1.0-1.noarch
bacon-proxy.jomitsch.example.com-qpid-client-cert-1.0-1.noarch
gssproxy-0.7.0-4.el7.x86_64
bacon-proxy.jomitsch.example.com-puppet-client-1.0-1.noarch
rubygem-smart_proxy_pulp-1.3.0-1.el7.noarch
bacon-proxy.jomitsch.example.com-foreman-proxy-client-1.0-1.noarch
bacon-proxy.jomitsch.example.com-qpid-router-server-1.0-1.noarch
bacon-proxy.jomitsch.example.com-qpid-broker-1.0-1.noarch
foreman-proxy-1.18.0-0.develop.201802282028gitaf41016.el7.noarch
bacon-proxy.jomitsch.example.com-foreman-proxy-1.0-1.noarch
bacon-proxy.jomitsch.example.com-qpid-router-client-1.0-1.noarch
```

Reply:
We resolved this outside of this thread, misunderstanding on my part

Reply:
@johnpmitsch, would it improve the readability if I renamed the installer_type symbols to `scenario`, `:legacy_katello` and `:legacy_capsule`. Or `:non_scenario_katello` and `:non_scenario_capsule`?

Reply:
@mbacovsky I think that would be a good idea. My vote goes to the `:legacy_` options